server/polar/kit/api/responses.py

```diff
@@ -0,0 +1,29 @@
+from typing import Any, Dict, Union
+
+from starlette.exceptions import HTTPException
+
+
+def get_api_errors(*additional_responses) -> Dict[Union[int, str], Dict[str, Any]]:
```

Comment:
This is not needed... 👀

Reply:
@Kludex thanks for taking a look 🤝 , how can we change this what do you suggest?

Reply:
Just create the dictionary on each endpoint

Reply:
Thanks @Kludex for the great feedback ♥️ I agree with it that we can simply add the responses directly to the router decorator, i.e no need for the `get_api_errors` function. @sultaniman Thanks for helping out with this and improving the documentation/Swagger of our endpoints – much appreciated 🙏🏼

Reply:
@Kludex @birkjernstrom then I will close this PR and will instead open a separate one.

Reply:
@sultaniman Awesome 🙏🏼 You can convert it to a draft too and ping me at any point if you want an earlier review too – whatever you prefer 🙂

Reply:
Oops. Saw now that you closed it. I was too slow :)